OpenTESArena/src/Game/Game.cpp

```diff
@@ -36,7 +34,8 @@ Game::Game()
 	this->options = OptionsParser::parse();
 
 	// Initialize virtual file system using the Arena path in the options file.
-	VFS::Manager::get().initialize(std::string(this->options->getArenaPath()));
+    std::string arena = File::toString(this->options->getArenaPath());
```

Comment:
Um... I'm not sure this is doing what you want it to do. `File::toString()` takes a filename and returns all of the content of the file. We don't want to initialize the virtual file system with the content of the `ARENA` filename because it's a folder.